public/static/oidc-callback.html

```diff
@@ -47,8 +47,19 @@
                 return oidcUserManager.signinRedirectCallback();
             })
             .then((user) => {
-                const redirectTo = user.state;
-                window.location.href = redirectTo && isUrlSaveForRedirect(redirectTo) ? redirectTo : "../";
+                    const redirectTo = user.state;
+                    // Verhindere Redirect auf Callback- oder Login-Seite
+                    const forbiddenRedirects = [window.location.pathname, '/login', 'oidc-callback.html'];
```

Comment:
Preventing a redirect to `/login` would break OIDC authentication entirely, since that is where the actual authentication with DT happens:

https://github.com/DependencyTrack/frontend/blob/8fd757be612eaf4f35eadbe4c334204d7bd711be/src/views/pages/Login.vue#L266-L281

Was this fix tested?